src/extractor/scripting_environment_lua.cpp

```diff
@@ -47,6 +49,11 @@ template <class T> double lonToDouble(T const &object)
     return static_cast<double>(util::toFloating(object.lon));
 }
 
+template <class T> osmium::Location locationForNode(T const &object)
+{
+        return LuaScriptingEnvironment::extraction_containers->getLocationForNode(object);
```

Comment:
I create this function and i pass it to luabind but into i can't acces to extraction_containers if it's not a static var : 
 error: invalid use of non-static data member ‘osrm::extractor::LuaScriptingEnvironment::extraction_containers’